dp-cluster-setup-utility.py

```diff
@@ -367,6 +368,16 @@ def _find_cluster(self):
     _, response = self.client.get((Url('clusters') / cluster_name).query_params(fields='Clusters/security_type,Clusters/version,Clusters/cluster_name'))
     return Cluster(response['Clusters'], self.client.rebased(self.base_url / 'api' / self.api_version / 'clusters' / cluster_name))
 
+  def _find_repository_version(self, cluster_name):
+    # Not able to use query_params in the line below because of the '/' in the key name.
+    _, response = self.client.get(Url((Url('clusters') / cluster_name / 'stack_versions').base + '?ClusterStackVersions/state=CURRENT'))
```

Comment:
I was thinking about this and realized that query_params() still can be used by passing the params in a dictionary and adding **.

For example

```python
Url('/whatever').query_params(**{ 'ClusterStackVersions/state': 'CURRENT' })
```